brave-spark-framework-filter/README.md

```diff
@@ -7,33 +7,13 @@ create and submit span with `sr`', `ss` annotations.
 
 Here is a use Example:
 
-Add a dependency:
+```java
 
-```
-<dependency>
-    <groupId>io.zipkin.brave</groupId>
-    <artifactId>brave-spark-framework-filter</artifactId>
-    <version>4.0.6</version>
-</dependency>
-        
-```
-
-Use it in Spark:
-
-```
-    Sender sender = OkHttpSender.create("http://127.0.0.1:9411/api/v1/spans");
-    
-    Reporter reporter = new LoggingReporter();
-    //Reporter reporter = AsyncReporter.builder(sender()).build();
-    
-    Brave brave = new Brave.Builder("service-name").reporter(reporter).build();
-    
-    Spark.before(BraveSparkRequestFilter.create(brave));
-    
+    Spark.before(BraveSparkFilter.create(brave).requestFilter());
     Spark.get("/foo", (req, res) -> "bar");
-    
+    //if you need add binaryAnnotation "error",use this.
```

Comment:
I'd still recommend always setting this (ex remove the clarification, or change it to "unless you add this, spans will not be marked as error).

Also, it is still sensible to have the configuration comments above the user code. It is more clear what is brave related and what is user related. In other words..

```java
// setup brave tracing...
Spark.before(braveTracing.before());
Spark.afterAfter(braveTracing.afterAfter());
Spark.exception(Exception.class, BraveSparkExceptionHandler.create(brave, new ExceptionHandlerImpl()));

// any routes you add are now traced, such as the below
 Spark.get("/foo", (req, res) -> "bar");
```